README.md

```diff
@@ -1,24 +1,132 @@
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Database initialization
+# 📖 BlogApp
 
-* How to run the test suite
+>This project is a blog where you'll find resources about rails technologies, you can leave likes and comment on posts and engage with the community. Built with Ruby on Rails
 
-* Services (job queues, cache servers, search engines, etc.)
+## 🛠 Built With <a name="built-with"></a>
 
-* Deployment instructions
+- `Ruby`
+- `Ruby on rails`
 
-* ...
+### Key Features <a name="key-features"></a>
+
+- Add post
+- Leave likes and comments on post
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Presentation Video<a name="live-demo"></a>
+
+- Not available
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started
+To clone the repository in local environment try following steps.
+
+```sh
+git clone 'https://github.com/hasnatali1947/BlogApp.git'
+```
+
+### Prerequisites
+- Rails
+- Ruby
+
+## Setup
+
+### Install
+
+In the terminal, go to the file directory and run this command.
+
+```sh
+bundle install
+```
+### Get into development
+- create database and run migrations
+
+```sh
+bin/rails db:setup
+```
+- start server
+
+```sh
+bin/rails server
+```
+
+### Run test
+To run test, run the following command in the terminal
+```sh
+rails test
+```
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Hasnat Ali**
+
+- GitHub: [Hasnat Ali](https://github.com/hasnatali1947)
+- LinkedIn: [ikenna_richard](https://www.linkedin.com/in/hasnattali/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Delete comments
+- Register to become a contributor
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome! Add suggestions by opening new issues.
+
+Feel free to check the [issues page](https://github.com/hasnatali1947/BlogApp.git).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- [Microverse](https://www.microverse.org/)
+- Coding Partners
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] The LICENSE link is broken, kindly make sure to fix it so all the links are functional in the README file